Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -29,11 +32,60 @@ class CameraViewControllerTests: XCTestCase {
         GiniConfiguration.sharedConfiguration.onboardingShowAtLaunch = false
         GiniConfiguration.sharedConfiguration.onboardingShowAtFirstLaunch = false
         
-        vc = CameraViewController(successBlock: { _ in }, failureBlock: { _ in })
         _ = vc.view
         
         XCTAssertFalse(vc.captureButton.isEnabled, "capture button should be disaled when tooltip is shown")
         
     }
+    
+    func testReviewButtonBackground() {
+        _ = vc.view
+        
+        let image = loadImage(withName: "invoice.jpg")
+        let imageData = UIImageJPEGRepresentation(image!, 0.9)
+        XCTAssertTrue(vc.multipageReviewBackgroundView.isHidden,
+                      "multipageReviewBackgroundView should be hidden before capture the first picture")
+        vc.cameraDidCapture(imageData: imageData, error: nil)
+        XCTAssertTrue(vc.multipageReviewBackgroundView.isHidden,
+                      "multipageReviewBackgroundView should be hidden after capture the first picture")
+        vc.cameraDidCapture(imageData: imageData, error: nil)
+        XCTAssertTrue(vc.multipageReviewBackgroundView.isHidden,
+                      "multipageReviewBackgroundView should not be hidden after capture the second picture")
+        
+    }
+    
+    func testReviewImagesButtonTouchInteraction() {
+        _ = vc.view
+        
+        let image = loadImage(withName: "invoice.jpg")
+        let imageData = UIImageJPEGRepresentation(image!, 0.9)
+        XCTAssertFalse(vc.multipageReviewButton.isUserInteractionEnabled,
+                       "multipageReviewButton should be disabled on start")
+        
+        vc.cameraDidCapture(imageData: imageData, error: nil)
+        
+        let predicate = NSPredicate(format: "isUserInteractionEnabled == YES")
+        
+        _ = self.expectation(for: predicate,
+                             evaluatedWith: vc.multipageReviewButton,
+                             handler: .none)
+        waitForExpectations(timeout: 1.5, handler: { result in
+            XCTAssertNil(result, "multipageReviewButton should be enabled after capturing a picture")
+            XCTAssertTrue(self.vc.multipageReviewBackgroundView.isHidden,
+                          "multipageReviewBackgroundView should be hidden before capture the first picture")
+            self.vc.cameraDidCapture(imageData: imageData, error: nil)
+            
+            let predicate = NSPredicate(format: "isHidden == NO")
+            
+            _ = self.expectation(for: predicate,
+                                 evaluatedWith: self.vc.multipageReviewBackgroundView,
+                                 handler: .none)
+            self.waitForExpectations(timeout: 1.5, handler: { result in
```

Comment:
This method has two expectations and 4 assertions. It tests a whole range of events and is therefore not a correct unit test.

Reply:
That is what I was expecting to discuss with you. I wasn't going to use this test in the future, because for me it is more a UITest, but since the UITests aren't fixed yet I just wanted to keep it there.
But maybe I forget about this, so I will remove it.